deploy/server-setup-guide.md

```diff
@@ -0,0 +1,118 @@
+# Server setup guide
+
+System and network requirements:
+- Ubuntu 18.04 LTS
+- Public static IP address
+
+### Prepeare system:
+#### 1. Upgrade system packages
+```sh
+sudo apt update
+sudo apt upgrade
+```
+#### 2. Install software dependencies
+```sh
+sudo apt install git python3 python3-pip nginx
+```
+#### 3. Install NodeJS v14
+```sh
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
+sudo apt install nodejs
+```
+
+### Set up gunicorn server
+#### 1. Clone the codebase
+```sh
+cd ~
+git clone https://github.com/WebUraXalys/blackout.git
+```
+#### 2. Select the desired branch
+```sh
+git switch <DESIRED BRANCH>
+```
+#### 3. Set up the virtual environment
+```
+python3 -m venv venv
```

Comment:
When run command to create environment. It gives the following error: 

`The virtual environment was not created successfully because ensurepip is not

available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/home/ubuntu/venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']
`

Reply:
Fixed this in last commit.